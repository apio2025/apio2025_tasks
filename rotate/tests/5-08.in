tXwyMZV19lL68RlQwHgOG0E5XaXD61x1
300
17436 4813 38015 460 7929 31872 17761 25141 3879 25179 36756 8351 29634 2532 39769 425 49300 35001 43454 13212 17119 19496 7788 14679 8153 19946 10111 34936 8149 35482 34755 38470 13674 899 1303 28123 5842 28071 38097 6703 1985 39164 8190 14544 19328 5728 4004 158 20917 17630 35318 22738 29281 18025 38787 36559 49909 47445 7963 41345 3154 7733 20879 11809 24172 21006 11668 48250 35178 12507 13060 22694 34508 2634 25649 19090 6425 16451 40502 7249 5109 40441 26556 14795 24603 41865 6678 39542 19756 18992 37482 41965 30020 22953 40285 37292 35174 14291 10297 38702 6564 14160 24269 34174 17885 5450 14257 361 34322 28609 26597 21269 14566 12400 11621 12347 45490 2381 35435 25503 20083 9038 10993 8640 19681 34681 16193 10075 29676 15988 35547 43290 40101 11836 49954 20858 45542 21089 40019 32282 7135 25277 3667 36719 49134 11477 17445 45045 5322 3608 6292 30527 38012 17205 47139 3311 22427 44346 4991 10018 19651 10343 3181 32434 34752 35040 23461 20447 45517 28545 41802 43224 34947 6508 22373 10419 44992 38702 23310 1388 28247 33055 13895 11930 445 25448 28308 26013 1191 29487 7610 29780 24061 33246 41267 31339 36635 16642 27836 44081 27492 2799 27963 5863 34163 6257 11920 4949 6896 16477 5350 35671 9724 12614 23750 11252 24025 40923 47385 2175 6504 47421 10440 35414 15312 28735 22498 6568 19096 40070 15449 7860 28385 33904 29433 23724 19763 34593 6374 904 15299 44376 31137 35991 39645 44800 31318 8001 15380 25810 44453 18328 23075 39816 4204 1276 12431 27900 12797 1901 43451 21935 28161 43323 9927 32870 17449 35162 2663 11855 27724 5330 14635 31905 44306 28818 28912 48977 37819 308 29810 49313 46349 46172 49932 44467 29545 25546 42686 843 4777 47999 13273 38258 9832 19275 24923 47943 8557 8373
