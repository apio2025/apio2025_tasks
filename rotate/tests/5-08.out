Kk1WryViMzuiNVd5LmhgkRlqQl5ntK4r
OK
